flink-formats/flink-protobuf/src/test/proto/test_simple.proto

```diff
@@ -22,28 +22,25 @@ option java_package = "org.apache.flink.formats.protobuf.testproto";
 option java_multiple_files = true;
```

Comment:
what about rename this file to test_simple_multi.proto so that the name convention is the same as timestamp?

Reply:
https://github.com/apache/flink/pull/21613#discussion_r1064118738